backend/src/game/PokerTable.ts

```diff
@@ -74,4 +87,17 @@ export default class PokerTable {
     public static getAllTables(): PokerTable[] {
         return Object.values(PokerTable.tables);
     }
+
+    public static checkTableReady(table: PokerTable): Result<void>{
```

Comment:
I think this can stay private and it doesn't need to be unit tested. Only public functions need to be unit tests, IE the caller: `sitAtTable`.

 It can also return a boolean instead as the error message isn't useful to the player or anyone (at least at this stage, maybe we would change this in the future)

Reply:
Ok cool. So I just won't have any unit tests for this feature then since it's a private function within the sit at table function.

Reply:
`sitAtTable` should still be unit tested and you can use most of what you have already I think!

Reply:
Apologies, yeah no unit tests are needed for this PR. I got my wires crossed when reviewing this PR. I will create a task for the unit tests for `sitAtTable` 👍

Reply:
Hey @richardaspinall so what result object should I use?

Do you mean just set the error message to empty ie return Result.error(''); or should I use ResultError('');?

Reply:
So no Result for this function. It would instead return a boolean:

```suggestion
    public static checkTableReady(table: PokerTable): boolean {
```

then:

- `return false` when there is an empty seat
- `return true` when no empty seats / the table is ready

Reply:
Then the `sitAtTable` function, and it reads well`.

```
if(tableReady) {

}
```

(even better if the variable was `tableIsReady` / "if table is ready")

Reply:
Ah ok sick. Thanks